var/spack/repos/builtin/packages/pmix/package.py

```diff
@@ -32,15 +32,15 @@ class Pmix(AutotoolsPackage):
     itself."""
 
     homepage = "https://pmix.org"
```

Comment:
Officially, the library's homepage is https://openpmix.github.io/. The homepage for the Standard is at https://pmix.github.io.

Reply:
Okay, fixed.